test/unit/mp_print_unknown_ext.c

```diff
@@ -1,4 +1,4 @@
-#include "msgpack.h"
+#include "box/msgpack.h"
 #include "mp_extension_types.h"
```

Comment:
Hmm, why we don't need similar change for:

```
src/box/box.cc:#include "msgpack.h"
src/box/msgpack.c:#include "msgpack.h"
```

Reply:
Because all of them (`box.cc`, `msgpack.c`, and `msgpack.h`) are located in the same directory.